index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link href="style.css" rel="stylesheet">
+  <body>
+    <header id="header">
+      <img src="" id="header-img">
+        <nav id="nav-bar" class="navbar navbar-expand-lg navbar-light bg-light">
+          <ul>
+          <li>
+            <a class="nav-link" href="#features">Features</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#how-it-works">How It Works</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#pricing">Pricing</a>
+          </li>
+        </nav>
```

Comment:
Please consider looking into this part of the code, where `</u>` is missing and an `indentation` issue.

Reply:
> Please consider looking into this part of the code, where `</u>` is missing and an `indentation` issue. 😅

Comment is polite. ✔️, however, let's always remember to explain why we request a change - it might be obvious for you but not for someone else. 💪🤗

Reply:
> Please consider looking into this part of the code, where `</u>` is missing and an `indentation` issue. 😅

Comment is polite. ✔️, however, let's always remember to explain why we request a change - it might be obvious for you but not for someone else. 💪🤗